packages/demos-babylon/src/chatbotDemo.html

```diff
@@ -8,12 +8,6 @@
 
     <link rel="stylesheet" href="chatbotDemo.css" />
 
-    <!-- Sumerian Host dependencies -->
-    <script
-      type="text/javascript"
-      src="https://sdk.amazonaws.com/js/aws-sdk-2.645.0.min.js"
```

Comment:
Is this deletion intended?

Reply:
yea, we bundle the SDK so this doesn't need to be on the page.